base/config-rzr.h

```diff
@@ -196,20 +196,6 @@ using namespace std;
 
 //--------------------------------------------------------------------------
 
-#ifdef _MSC_VER
```

Comment:
did you try to build it under msvc ?
if not please preserve this change , 
and please commit a cleanup patch separately

Reply:
I haven't tried with Windows and Visual so I removed cleanup changes as suggested.